tests/test_globals.hpp

```diff
@@ -31,10 +31,11 @@ struct print_log_value<glm::vec3> {
         s << glm::to_string(v);
     }
 };
+BOOST_NS_MAGIC_CLOSING
 }
 }
-BOOST_NS_MAGIC_CLOSING
 
+#if BOOST_VERSION < 106400
```

Comment:
Please check and fix this check or mark this comment as a todo. I'd go as far and call this a bug otherwise ;)